# DP: Backport r241171 from trunk

syscall: mark rawClone as no_split_stack

Reviewed-on: https://go-review.googlesource.com/30955

Index: b/libgo/go/syscall/clone_linux.c
===================================================================
--- a/libgo/go/syscall/clone_linux.c	(revision 241071)
+++ b/libgo/go/syscall/clone_linux.c	(revision 241072)
@@ -10,7 +10,10 @@
 
 #include "runtime.h"
 
-long rawClone (unsigned long flags, void *child_stack, void *ptid, void *ctid, struct pt_regs *regs) __asm__ (GOSYM_PREFIX "syscall.rawClone");
+long rawClone (unsigned long flags, void *child_stack, void *ptid,
+	       void *ctid, struct pt_regs *regs)
+  __asm__ (GOSYM_PREFIX "syscall.rawClone")
+  __attribute__ ((no_split_stack));
 
 long
 rawClone (unsigned long flags, void *child_stack, void *ptid, void *ctid, struct pt_regs *regs)
